cmd/podman/cp.go

```diff
@@ -0,0 +1,244 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/buildah/util"
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/pkg/chrootuser"
+	"github.com/containers/storage"
+	"github.com/containers/storage/pkg/archive"
+	"github.com/containers/storage/pkg/chrootarchive"
+	"github.com/containers/storage/pkg/idtools"
+	digest "github.com/opencontainers/go-digest"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	cpDescription = "Copy files/folders between a container and the local filesystem"
+	cpCommand     = cli.Command{
+		Name:         "cp",
+		Usage:        "Copy files/folders between a container and the local filesystem",
```

Comment:
Why to not reuse `cpDescription` here too? like the line below instead of repeat the same string.

Eventually I guess you need to rewrite your usage with something like:
```go
Usage:      "podman cp [CONTAINER:]SRC_PATH [CONTAINER:]DEST_PATH"
```
Thoughts?

Reply:
After testing your generated output seem good:
```
bin/podman cp -h
NAME:
   podman cp - Copy files/folders between a container and the local filesystem

USAGE:
   podman cp [CONTAINER:]SRC_PATH [CONTAINER:]DEST_PATH

DESCRIPTION:
   Copy files/folders between a container and the local filesystem
```

Reply:
yes, cli.Command.Usage goes to NAME finally :wink: